doc/faq/virtualenv_faq.rst

```diff
@@ -40,6 +43,7 @@ exist. Some of these are given here:
 * The ``TKAgg`` backend doesn't require any external dependencies and is
   normally always available.
 * The ``QT4`` framework ``PySide`` is pip installable.
+* ``PYQT5`` is pip installable on Python 3.5.
 * The upcoming `WX Phoenix <http://wiki.wxpython.org/ProjectPhoenix>`_ toolkit
   is ``pip`` installable.
```

Comment:
Agreed, This was originally written under the assumption that a release of WX Phoenix was to happen soon. Evidently that is not the case. We can add it again once the release happens